README.md

```diff
@@ -1,13 +1,32 @@
-# Mongo API Project
+# City of birches
```

Comment:
My first thought when I saw your data was "F@$#, do day have different species than Betulas??"  You played on my sentiment here 😸 You know that I was study Gardening for 1 year in Poland :D  Dendrology was my favorite classes 🌳